graf2d/gpadv7/src/RFrame.cxx

```diff
@@ -55,12 +56,12 @@ void RFrame::PopulateMenu(RMenuItems & /* items */)
    auto is_y = items.GetSpecifier() == "y";
 
    if (is_x || is_y) {
-      RAttrAxis &attr = is_x ? AttrX() : AttrY();
-      std::string name = is_x ? "AttrX()" : "AttrY()";
+      RAttrAxis &attr = is_x ? x : y;
+      std::string name = is_x ? "x" : "y";
```

Comment:
Following up the comment above, on the coding convention of these special member variables, we discussed writing this differently to clearly signal member variables as such:
```suggestion
      RAttrAxis &attr = is_x ? this->x : this->y;
      std::string name = is_x ? "x" : "y";
```